scripts/fetch_and_build_deb

```diff
@@ -49,15 +49,25 @@ fi
 # populate variables from packaging metadata file
 # shellcheck source=/dev/null
 source /buildfiles/pkgvars
+# populate pg versions to be used for packaging.
+# supported-potgres file is derived file from postgres-matrix.yml file
+source /buildfiles/supported-postgres 2>/dev/null
```

Comment:
This will fail if the file does not exist. Redirecting stderr does not prevent nonzero vales returned by `source`

Reply:
![image](https://user-images.githubusercontent.com/1596331/135808440-ee5e57e1-4d95-4114-adf0-20f78b235f3d.png)

Reply:
You can check exit code of the last command by `echo $?`

Alternatively, you can use `set -euo pipefail` in bash to have a similar setting on your terminal.

Not having any output on your screen does not guarantee that the command ran successfully.

Reply:
You're right I added file existence check now